assets/js/modules/adsense/components/settings/SettingsView.js

```diff
@@ -152,26 +165,64 @@ export default function SettingsView() {
 			{ webStoriesActive && (
 				<div className="googlesitekit-settings-module__meta-items">
 					<div className="googlesitekit-settings-module__meta-item">
-						<div className="googlesitekit-settings-module__meta-item">
-							<h5 className="googlesitekit-settings-module__meta-item-type">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __( 'Web Stories Ad Unit', 'google-site-kit' ) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							{ ! webStoriesAdUnit && (
+								<span>{ __( 'None', 'google-site-kit' ) }</span>
+							) }
+							{ webStoriesAdUnit && (
+								<DisplaySetting value={ webStoriesAdUnit } />
+							) }
+						</p>
+					</div>
+				</div>
+			) }
+
+			{ !! adBlockingRecoverySetupStatus?.length && (
+				<div className="googlesitekit-settings-module__meta-items">
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __( 'Ad blocking recovery', 'google-site-kit' ) }
+						</h5>
+						{ adBlockingRecoverySetupStatus ===
+							'setup-confirmed' && (
+							<p className="googlesitekit-settings-module__meta-item-data">
 								{ __(
-									'Web Stories Ad Unit',
+									'Ad blocking recovery tag is not placed',
 									'google-site-kit'
 								) }
-							</h5>
-							<p className="googlesitekit-settings-module__meta-item-data">
-								{ ! webStoriesAdUnit && (
-									<span>
-										{ __( 'None', 'google-site-kit' ) }
-									</span>
-								) }
-								{ webStoriesAdUnit && (
-									<DisplaySetting
-										value={ webStoriesAdUnit }
-									/>
-								) }
 							</p>
-						</div>
+						) }
+						{ adBlockingRecoverySetupStatus === 'tag-placed' && (
+							<Fragment>
+								<p className="googlesitekit-settings-module__meta-item-data">
+									<strong>
+										{ __(
+											'Ad blocking recovery tag is placed',
+											'google-site-kit'
+										) }
+									</strong>
```

Comment:
Unfortunately I think we need to deviate from the design a bit here (it's already not consistent with what we have now which is the problem) as all other setting values are not bold – with the current design this looks the same as a heading.

<img width="301" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/e8b62c29-9fb0-4c01-9e79-8cdfc9156f10">

Let's remove the bold for consistency, and note this difference for QA

Reply:
Cool, will do 👍🏻